starknet_py/utils/merkle_tree.py

```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass, field
+from typing import List, Tuple
+
+from starknet_py.hash.hash_method import HashMethod
+from starknet_py.net.client_utils import _to_rpc_felt
+
+
+@dataclass
+class MerkleTree:
+    """
+    Dataclass representing a MerkleTree object.
+    """
+
+    leaves: List[str]
```

Comment:
Why do we store the leaves as `str` instead of `int`?

Reply:
I wanted to store them as `str` representing felt values. However, considering that hashing methods use `int` it may be more suitable to rely on mentioned `int`. Also, it reduces extra conversions. Changes applied 👍